packages/application-config/src/validate-config.ts

```diff
@@ -9,6 +9,8 @@ type ErrorAdditionalProperty = ErrorObject<
 type ErrorEnum = ErrorObject<'enum', { allowedValues: string[] }>;
 
 const ajv = new Ajv({ strict: true, useDefaults: true });
+require('ajv-keywords')(ajv, 'uniqueItemProperties');
```

Comment:
Hmm I'm not sure we can 100% rely on this validation at the schema level, especially when values are provided using environment variables (env placeholders).

I think we need to validate these kind of things after parsing the config and resolving the values. For instance, in the `transformCustomApplicationConfigToData`.

Reply:
Oh. I didn't think of that. 
I believe it makes more sense to do the validation in `transformCustomApplicationConfigToData `then`.